pythonforandroid/bootstraps/common/build/src/main/java/org/kivy/android/DashCamPythonService.java

```diff
@@ -0,0 +1,212 @@
+package org.kivy.android;
+
+import android.os.Build;
+import java.lang.reflect.Method;
+import java.lang.reflect.InvocationTargetException;
+import android.app.Service;
+import android.os.IBinder;
+import android.os.Bundle;
+import android.content.Intent;
+import android.content.Context;
+import android.util.Log;
+import android.app.Notification;
+import android.app.PendingIntent;
+import android.os.Process;
+import java.io.File;
+
+//imports for channel definition
+import android.app.NotificationManager;
+import android.app.NotificationChannel;
+import android.graphics.Color;
+
+public class DashCamPythonService extends Service implements Runnable {
+
+    private static final String SERVICE_ENTRY_POINT= "gen3cam/services/dashCamUploaderService.py";
+    private static final String PYTHON_NAME = "dashCamUploaderService";
+
+    // Thread for Python code
+    private Thread pythonThread = null;
+
+    // Python environment variables
+    private String androidPrivate;
+    private String androidArgument;
+    private String pythonName;
+    private String pythonHome;
+    private String pythonPath;
+    private String serviceEntrypoint;
+    // Argument to pass to Python code,
+    private String pythonServiceArgument;
+
+
+    public static DashCamPythonService mService = null;
```

Comment:
Do we really need a reference to itself as if it was a singleton which is not?

Reply:
This is a copy cat of [PythonService](https://github.com/sriramaravamuthan/python-for-android/blob/develop/pythonforandroid/bootstraps/common/build/src/main/java/org/kivy/android/PythonService.java)